### PR TITLE
fix: allow for specifying x-fern-examples as the yaml schema, not jus…

### PR DESCRIPTION
**Pull request by armandobelardo**
…t the bloated EndpointExample schema

### CODE REVIEW

File: packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml [hunk 1]
```diff
@@ -196,6 +196,11 @@ types:
       - jvm # alias to java
 
   EndpointExample:
+    union:
+      unknown: unknown
```
Comment (dsinghvi) [inline packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml:200]: Can we create an alias `FernExample`

**Comment by dsinghvi** [comment]
Can we add a snapshot test for this?

File: packages/cli/openapi-parser/src/openapi/v3/hasIncompleteExample.ts [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+        return (
+            example.request == null &&
+            example.response == null &&
+            example.queryParameters == null &&
+            example.pathParameters == null &&
+            example.headers == null
+        );
+    } else {
+        const castExample = example.value as RawSchemas.ExampleEndpointCallSchema;
+        // throw new Error(`Here we are: ${JSON.stringify(castExample)}`);
```
Comment (dsinghvi) [inline packages/cli/openapi-parser/src/openapi/v3/hasIncompleteExample.ts:16]: nit: we'll want to delete but makes sense why you added

**Review by dsinghvi — APPROVED** [review]

File: packages/cli/openapi-parser/src/openapi/v3/hasIncompleteExample.ts (hunk 2 above)
Comment (armandobelardo) [reply packages/cli/openapi-parser/src/openapi/v3/hasIncompleteExample.ts:16]: haha whoops

File: packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml (hunk 1 above)
Comment (armandobelardo) [reply packages/cli/openapi-ir-sdk/fern/definition/finalIr.yml:200]: Done!

File: packages/cli/generation/ir-generator/src/__test__/__snapshots__/openapi/code-samples-open-api.txt [hunk 3]
```diff
@@ -144,51 +144,7 @@ exports[`generate IR from OpenAPI 1`] = `
...
-};
-axios.request(options).then(function (response) {
-  console.log(response.data);
-});",
-                  "docs": null,
-                  "name": null,
-                  "sdk": "typescript",
-                  "type": "sdk",
-                },
-              ],
```
Comment (dsinghvi) [inline packages/cli/generation/ir-generator/src/__test__/__snapshots__/openapi/code-samples-open-api.txt:191]: @armandobelardo this is weird, why is this happening? some customers rely on this functionality

Comment (armandobelardo) [reply packages/cli/generation/ir-generator/src/__test__/__snapshots__/openapi/code-samples-open-api.txt]: Fixed!

**Pull request merged**